ci/travis-36.yaml

```diff
@@ -8,6 +8,8 @@ dependencies:
   - dask
   - fastparquet
   - feather-format
+  - flake8
```

Comment:
I think we have a reference in the docs to our version of supported flake. This should be >= 3.5?

Reply:
Does seem like we have any reference. A grep to the whole source code just shows me references to flake8 in this section: https://pandas.pydata.org/pandas-docs/stable/contributing.html#python-pep8

And I don't think now it makes sense to add it. Before the change flake8 had to be `3.4.1` (or older), or the linting failed. Now, the linting should work no matter the version of flake8. Older versions check less errors, but that's a detail of flake8, not about pandas.

Reply:
better to actually have this requirement